packages/turf-concave/index.js

```diff
@@ -31,40 +32,41 @@ var distance = require('@turf/distance');
  * //addToMap
  * var addToMap = [points, hull]
  */
-function concave(points, maxEdge, units) {
-    if (typeof maxEdge !== 'number') throw new Error('maxEdge parameter is required');
+module.exports = function (points, maxEdge, units) {
+    if (!points) throw new Error('points is required');
+    if (maxEdge === undefined || maxEdge === null) throw new Error('maxEdge is required');
+    if (typeof maxEdge !== 'number') throw new Error('invalid maxEdge');
 
     var tinPolys = tin(points);
-    var filteredPolys = tinPolys.features.filter(filterTriangles);
-    tinPolys.features = filteredPolys;
-    if (tinPolys.features.length < 1) {
-        throw new Error('too few polygons found to compute concave hull');
-    }
-
-    function filterTriangles(triangle) {
+    tinPolys.features = tinPolys.features.filter(function (triangle) {
         var pt1 = triangle.geometry.coordinates[0][0];
         var pt2 = triangle.geometry.coordinates[0][1];
         var pt3 = triangle.geometry.coordinates[0][2];
         var dist1 = distance(pt1, pt2, units);
         var dist2 = distance(pt2, pt3, units);
         var dist3 = distance(pt1, pt3, units);
         return (dist1 <= maxEdge && dist2 <= maxEdge && dist3 <= maxEdge);
-    }
+    });
 
-    return merge(tinPolys);
-}
+    if (tinPolys.features.length < 1) throw new Error('too few polygons found to compute concave hull');
 
-function merge(polygons) {
-    var merged = JSON.parse(JSON.stringify(polygons.features[0])),
-        features = polygons.features;
+    return merge(tinPolys.features);
+};
+
+/**
+ * Merges/Unifies all the features in a single polygon
+ *
+ * @private
+ * @param {Array<Feature>} features to be merged
+ * @returns {Feature<(Polygon|MultiPolygon)>} merged result
+ */
+function merge(features) {
+    var merged = clone(features[0]);
+    merged.properties = {};
 
     for (var i = 0, len = features.length; i < len; i++) {
         var poly = features[i];
-        if (poly.geometry) {
-            merged = union(merged, poly);
-        }
+        if (poly.geometry) merged = union(merged, poly);
```

Comment:
Just an FYI, we are doing two `clone` operation here, one at the very top (which is fine) and a 2nd time internally with `union`. No need to change this, but we should one day revamp `union` (can be discussed separately).